src/theme/links.ts

```diff
@@ -4,6 +4,7 @@ const links = {
   components: {
     Link: {
       baseStyle: {
+        // If the link color(currently brand.c) is changed, search for all the occurances of the color used and update all instances
```

Comment:
Change comment to "If changing this line, also update link color in MarkdownStyler.module.css"